includes/utils/class-amp-image-dimension-extractor.php

```diff
@@ -2,20 +2,44 @@
 
 class AMP_Image_Dimension_Extractor {
 	static $callbacks_registered = false;
+    const STATUS_FAILED_LAST_ATTEMPT = 'failed';
+    const STATUS_IMAGE_EXTRACTION_FAILED= 'failed';
 
-	static public function extract( $url ) {
+    static public function extract( $urls ) {
```

Comment:
You're welcome! I know we talked about it on Slack, but just to close the loop here in case anyone else has been following along, I pushed these changes last week.